lib/mayaUsd/nodes/hdImagingShape.cpp

```diff
@@ -207,7 +207,7 @@ MObject PxrMayaHdImagingShape::GetOrCreateInstance()
     // We never intend for the imaging shape to get saved out to the Maya scene
     // file, so make sure that we preserve the scene modification status from
     // before we create the shape.
-    const UsdMayaBlockSceneModificationContext blockModContext;
+    const MayaUsd::utils::UsdMayaBlockSceneModificationContext blockModContext;
```

Comment:
Simply a reminder of internal discussion - let's remove the `UsdMaya` prefix since it's not very useful when the class now lives in the proper namespace.